ghcloneall.py

```diff
@@ -824,7 +824,7 @@ def _main():
         help='exclude archived repositories (default)')
     parser.add_argument(
         '--include-private', action='store_true', default=None,
-        help='include private repositories (default)')
+        help='include private repositories, requires a github token (default)')
```

Comment:
I apologize for highlighting the text in the README, when this is the location where it matters most.

(In other projects I started setting up tox environments to automatically check/update the --help message in the README with cog.  I plan to do this here too, I just haven't gotten around to it.)

Reply:
No problem, I should have caught that as I was making the update but I was focused on the wording. I pushed a new version with matching text.